app/views/comments/new.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_with( url: post_comments_path(@post), method: :post) do |form| %>
+  <%= form.label :text %>
+  <%= form.text_area :text %>
+
+  <%= form.submit 'Add Comment' %>
+<% end %>
```

Comment:
## Required

- [ ] Kindly make this form to be displayed as in the wireframes. It is required to accept input for creating comments